src/main/java/org/finance_price_service/domain/OneDayPrice.java

```diff
@@ -0,0 +1,61 @@
+package org.finance_price_service.domain;
+
+import java.io.Serializable;
+import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+import org.finance_price_service.domain.OneDayPrice.OneDayPriceId;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.format.annotation.DateTimeFormat;
+
+@Entity
+@Table(name = "stock_prices")
+@IdClass(OneDayPriceId.class)
+public class OneDayPrice{
```

Comment:
use @Value and @Builder lombok annotations on this class:

https://projectlombok.org/features/Builder
https://projectlombok.org/features/Value

@Value will take care of getters. @Builder allows you to create the object like this:

OneDayPrice oneDayPrice = OneDayPrice.builder().symbol().date()... etc... .build();

Reply:
I checked about @value and it's said that since lombok 1.16.0, constructor provided by @value is private. Since this class is an entity and a public constructor is needed, I need to also use @AllArgsConstructor @NoArgsConstructor. Otherwise, I could use @Getter and @Setter instead of @value.